src/utils/sidebarConfig.ts

```diff
@@ -258,6 +258,12 @@ function getCompleteSidebar(): SidebarConfig {
           schemaName: 'Party',
           filters: { role: 'Both' },
         },
+        {
+          label: t`Lead`,
+          name: 'lead',
+          route: '/list/Lead',
+          schemaName: 'Lead',
+        },
```

Comment:
move `Lead` to `Sales` section. Also set a feature flag for it to enable it on [AccountingSettings.ts](https://github.com/AbleKSaju/books/blob/3e3750da1e2470f5dafd7748fb35f480bf2079a3/models/baseModels/AccountingSettings/AccountingSettings.ts)